Src/Support/Google.Apis.Auth.AspNetCore3.IntegrationTests/Controllers/HomeController.cs

```diff
@@ -104,19 +104,27 @@ public async Task<IActionResult> StorageBucketListing([FromServices] IGoogleAuth
             return View(bucketNames);
         }
 
-        // Test showing use of incremental auth.
-        // The call to auth.RequireScopesAsync(...) ensures the correct authorization is available.
+        /// <summary>
+        /// Translates some text.
+        /// Specifying the <see cref="AuthorizeAttribute"/> will guarantee that the code executes only if the
+        /// user is authenticated.
+        /// No scopes are required via attributes.
+        /// Instead, scope are required via code using <see cref="IGoogleAuthProvider.RequireScopesAsync(string[])"/>.
+        /// </summary>
+        /// <param name="auth">The Google authorization provider.
+        /// This can also be injected on the controller constructor.</param>
         [Authorize]
-        public async Task<IActionResult> Translate([FromServices] IGoogleAuthProvider auth, [FromServices] ClientInfo clientInfo)
+        public async Task<IActionResult> Translate([FromServices] IGoogleAuthProvider auth)
         {
-            // Programmatic auth check.
+            // Check if the required scopes have been granted. 
             if (await auth.RequireScopesAsync(TranslateService.Scope.CloudTranslation) is IActionResult authResult)
             {
-                // If the required scopes are not authorized, then a non-null IActionResult will be returned,
-                // which must be returned from the action.
+                // If the required scopes are not granted, then a non-null IActionResult will be returned,
+                // which must be returned from the action. This triggers incremental authorization.
+                // Once the user has granted the scope, and authomatic redirect to this action will be issued.
```

Comment:
```suggestion
                // Once the user has granted the scope, an automatic redirect to this action will be issued.
```

(While I do love the idea of the word "authomatic" for automatic authorization, I think it's wishful thinking at this point :)

Reply:
Hahahaha, that would be a nice one :)